### DISCUSSION TITLE
FM Narrow-Wide

### DISCUSSION THREAD

**Discussion by Hunterfnet** [egzumer/uv-k5-firmware-custom#215] [Ideas]
It would be useful to be able to change FM narrow wide without having to access the main menu and be able to see the current mode in anticipation of possible future updates

**Comment by ErikS-web** [comment]
For what purpose ? 

The radio is a portofoon and most times used with pre-defined/fixed/memory channels.

Or do you mean during the spectrum ?

**Reply by Hunterfnet** [reply]
On same way that appears AM FM USB L M H C Ps SCR (Scanlist I II) VOX DW .... etc . A missused screen is allways an incomplete screen . Imagine a few frequency steps (for now only 2 N & W that only shows N and only at times) and possible 3 or more in near future (12.5 PMR , 25LPD , 5 CB VHF UHF... etc)  for me is most importat that for example that show a Ps on screen and more .

**Reply by ErikS-web** [reply]
For what purpose ?

**Reply by Hunterfnet** [reply]
if you don't understand me maybe i'm not on same level as you on communications , who care as you wish , i know the result with time.
For what purpose ? .. the upper the main important for be elegant and clean pannel and the lower ones more for a practical motives: I leave the theme ended , not need lost more time for my part , others it will lose for sure 1000% in future , dont bet for it.
CB Band 8khz ULTRA NARROW.... try insert 27.555 (Upper channels go in 5Khz 27.550...60) correctly and fast for ear good a DX call channel in USB (555), you can hear an approximated call with actual config,  but not a correct clear channel/frequency  . And much more impossible is save it with correct bandwith.

VHF UHF 12.5 Khz Narrow from 2013
152.0075, 157.4500 ,150.830, 150.920, 151.070, 151.190, 151.310, 152.480, 157.740, 158.460, 462.750, 462.775, 
462.800, 462.825, 462.850, 462.875, 462.900, 462.925, and 465.000 MHz 25Khz ... and a lot more .... SATELLITES ... etc

How do you save a correct frequency on memory ,fast if you haven't the correct info on channel screen ? Or more easy ... how you can hear a clear call if you can't select the correct bandwith ? Have a nice day .

**Reply by brookbonde** [reply]
Hunterfnet, I agree entirely with your post.  The operation of this radio is cumbersome and inconvenient at best using the clunky long winded menu.  It takes an age with many button presses required to achieve simple changes that should be available direct from the keypad.  The menu has too many items listed, and should be slimmed down or tidied up into sub menus.  It’s not a nice experience at all.  I’m not sure if it can be made to work any better or not.  

I think IJV firmware may be working more to improve the things that matter the most.  For example, that firmware already allows changing between FM AM USB by pressing a single button on the keypad, something I don’t believe is available on this firmware, but should be (please correct me if I’m wrong).  Have you tried that firmware at all?  Everyone I talk to about it says that IJV is better for the things that really matter when using the radio, although that firmware doesn’t have the scope, and never will have apparently (this is probably due to the limitation of space for the firmware).

Apparently FMwide is only supported by the BK1080 (according to this discussion board), so presumably that means FMW would only be available within the frequency range of the BK1080. I recently asked about this FMW option in another post.

**Reply by Hunterfnet** [reply]
Thanks brook , at times i think i'm inside a politic meeting full with fanatics sheeps . You maked my day . Don't confuse the FM wide (broadcast station) with the N, W .... modes , the amplitude of frequency , on this case and on this firmware , are for now only 2 and only shows one the N and can have 3 or more modes for correct transceiver use. The broadcast FM have a very big WIDE freq amplitude maybe an hipotetic W4 mode , i speak about a normal transceiver operation , i miss more FM bandwith modes bcz i like operate and receive on correct way and for now is impossible and i see comments here about have no audio change receiving in USB and changing a little freq or amplitude have the same result on this radio and i thinks myself .. it's so obvious , for you hear one same station and on real have 3 stations separated with 5 Khz up and down and your're on the middle 4 example .... Really thanks i agree  .

**Reply by Hunterfnet** [reply]
opps about the firmware IJV ... sincerely i dunno , i was starting on my UV5R with the official firmware V1.006 i think and just changed for test various firmwares after i saved a firm backup and my bad surprise i heard distortions on all frequencies and FM really bad until i just installed the egzumer all turns to normality and still here , i will test more on other UVK5 and compare firmwares and new versions but for now is all i can say ...

**Comment by f5gkw** [comment]
I agree 100/100 !
I am mainly a user of the IJV firmware for the reasons you relate.
And, IMHO, the " bandscope" is useless as it is possible to scan between 2
frequencies entered via the keypad, and blacklist one or several annoying
frequencies on the fly just by press the * key.
Actually my dream is to have a firmware capable to react when receive one (
of a list, why not) capcaode or RIC in the postage format, as postage is
used by the dapnet network.
Maybe by hacking the ccir (?)
I would like to write or modify a firmware by lyself, but I don't know how
to start to learn ....
73
JPhi " F5GKW".

Le dim. 10 déc. 2023 à 09:23, brookbonde ***@***.***> a
écrit :

**Reply by gartnl** [reply]
The bandscope is what I use most. There is a big difference between scanning and using the bandscope. The latter will respond to any signal in range that comes up. It's like superfast scanning. 
With the bandscope I can set up a large part of the spectrum and be allerted to any transmission immediatly.

**Comment by f5gkw** [comment]
Bad automatic corrector.
Please read POCSAG in place of postage...

**Comment by Hunterfnet** [comment]
i got yesterday the cause of the bad reception / transmission on amplitude  : The Beken BK4819 chip support 4 FM spacing modes 6.25 / 12.5 (N) / 20 / 25 (W) kHz , not sure if now are working 2 only on current firmware: N and W modes only , that's why we have problems hearing and saving channels correctly . 2 modes more aren't supported on software for now and causes interferences (overlapping) with near transmitting stations more when transmitting in USB . I think i will start testing the IJV ending this  week for compare on weekend
